pkg/resources/prometheus/configmap-rules.go

```diff
@@ -256,6 +256,13 @@ groups:
     labels:
       kubermatic: federate
 
+- name: kubermatic.kubernetes.apiserver
```

Comment:
please delete this `- name: kubermatic.kubernetes.apiserver`. No need for a dedicated job we already have one, `kubermatic.etcd`.